charts/trino/values.yaml

```
@@ -285,21 +285,38 @@ service:
   port: 8080
 
 auth: {}
-# auth -- Available authentication methods.
-# @raw
-# Use username and password provided as a [password file](https://trino.io/docs/current/security/password-file.html#file-format):
-# ```yaml
-#  passwordAuth: "username:encrypted-password-with-htpasswd"
-# ```
-# Set the name of a secret containing this file in the password.db key
-# ```yaml
-#  passwordAuthSecret: "trino-password-authentication"
-# ```
-# Additionally, set [users' groups](https://trino.io/docs/current/security/group-file.html#file-format):
-# ```yaml
-#  refreshPeriod: 5s
-#  groups: "group_name:user_1,user_2,user_3"
-# ```
+  # auth -- Available authentication methods.
+  # Set the type of authentication "ldap" or "file" with authMethod
```

You have to keep the `@raw` annotation before the first example code block
```suggestion
  # Set the type of authentication "ldap" or "file" with authMethod
  # @raw
```